lib/tiappxml.js

```diff
@@ -810,6 +823,25 @@ function toJS(obj, doc) {
 					}
 					break;
 
+				case 'webpack':
+					const webpack = obj.webpack = {};
+					xml.forEachElement(node, elem => {
+						switch (elem.tagName) {
+							case 'type': {
+								webpack[elem.tagName] = xml.getValue(elem);
```

Comment:
what are the possible types? I see 'alloy' in the test/example. What others would there be? Do we need any sort of validation here?

Reply:
Valid types are

- alloy
- classic
- vue
- angular

These values will be verified by `appcd-plugin-webpack` when starting the Webpack build. I don't think we should do any additional validation here since that means we would have to maintain multiple places for validation if this list ever changes.